huaweicloud/services/acceptance/dws/data_source_huaweicloud_dws_disaster_recovry_tasks_test.go

```diff
@@ -0,0 +1,100 @@
+package dws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDwsDisasterRecoveryTasksDataSource_basic(t *testing.T) {
+	resourceName := "data.huaweicloud_dws_disaster_recovery_tasks.name_filter"
+	dc := acceptance.InitDataSourceCheck(resourceName)
+	name := acceptance.RandomAccResourceName()
+	password := acceptance.RandomPassword() + "a"
```

Comment:
The meaning of the character'a' is not clear. It is recommended that you change it and add a remark to specify that the password must contain at least 12 characters.

Reply:
done